ports/geckolib/properties.mako.rs

```diff
@@ -1077,6 +1077,28 @@ fn static_assert() {
 
 </%self:impl_trait>
 
+<%self:impl_trait style_struct_name="Column"
+                  skip_longhands="column-width">
+
+    fn set_column_width(&mut self, v: longhands::column_width::computed_value::T) {
+        use gecko_bindings::structs::nsStyleUnit;
+
+        match v.0 {
+            Some(au) => {
+                self.gecko.mColumnWidth.mUnit = nsStyleUnit::eStyleUnit_Coord;
+                unsafe { *self.gecko.mColumnWidth.mValue.mInt.as_mut() = au.0; }
```

Comment:
We have a few helpers in `values.rs` to deal with `nsStyleCoord`s, I think this could use them.